WordPress/src/test/java/org/wordpress/android/ui/reader/repository/usecases/GetDiscoverCardsUseCaseTest.kt

```diff
@@ -0,0 +1,102 @@
+package org.wordpress.android.ui.reader.repository.usecases
+
+import com.nhaarman.mockitokotlin2.anyOrNull
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions.assertThat
+import org.json.JSONArray
+import org.json.JSONObject
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.anyBoolean
+import org.mockito.ArgumentMatchers.anyLong
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.TEST_DISPATCHER
+import org.wordpress.android.datasets.ReaderDiscoverCardsTableWrapper
+import org.wordpress.android.datasets.wrappers.ReaderPostTableWrapper
+import org.wordpress.android.fluxc.utils.AppLogWrapper
+import org.wordpress.android.models.discover.ReaderDiscoverCard.InterestsYouMayLikeCard
+import org.wordpress.android.models.discover.ReaderDiscoverCard.ReaderPostCard
+import org.wordpress.android.test
+import org.wordpress.android.ui.reader.ReaderConstants
+
+@InternalCoroutinesApi
+@RunWith(MockitoJUnitRunner::class)
+class GetDiscoverCardsUseCaseTest {
+    private lateinit var useCase: GetDiscoverCardsUseCase
+    private val readerDiscoverCardsTableWrapper: ReaderDiscoverCardsTableWrapper = mock()
+    private val parseDiscoverCardsJsonUseCase: ParseDiscoverCardsJsonUseCase = mock()
+    private val mockedJsonArray: JSONArray = mock()
+    private val mockedJsonObject1: JSONObject = mock()
+    private val mockedJsonObject2: JSONObject = mock()
```

Comment:
Just a suggestion here, I understood the tests clearly with these values but I think if we gave these objects some more meaningful names they would probably be easier to read. Something like `InterestsCardJson` and `ReaderPostCardJson`. Let me know what you think, you really don't have to make this change because we are mocking JSON objects so the connotations here are fine 🙏

Reply:
👍 Agree - fixed in 66f7a9314c5d6d620387ca60ff34466f69fb0f42